app/views/favorites/index.html.erb

```diff
@@ -1,17 +1,34 @@
 <h1>Favorite Pets Index</h1>
+<section class="index">
```

Comment:
Added styling to the favorites index page. I hope you don't mind! I was perusing the site making sure everything was displaying properly and couldn't help it!

Reply:
I would have liked to work on the styling. I think I want to practice more on that, and I felt like that's what we had discussed during the check-in.